pysatModels/tests/test_utils_extract.py

```diff
@@ -0,0 +1,579 @@
+"""Unit tests for `pysatModels.utils.extract`."""
+
+from io import StringIO
+import logging
+import numpy as np
+from packaging import version as pack_version
+import pytest
+
+import pysat
+from pysat.instruments import pysat_testmodel
+
+import pysatModels as ps_mod
+import pysatModels.utils.extract as extract
+
+
+@pytest.mark.skip("input requires a regular grid for the model")
```

Comment:
I think we have a regular model now. This may need to be updated to a pysat version check.

Reply:
Really?  How do we call it?